### PR TITLE
Feat/decimal markers

### PR DESCRIPTION
**Pull request by JarbasAl**
port https://github.com/MycroftAI/lingua-franca/pull/69

## Summary by CodeRabbit

- **New Features**
  - Introduced a `normalize_decimals` function for handling different decimal characters in number extraction.
  - Updated number extraction functions to accept a custom decimal character.

- **Tests**
  - Added tests to validate number extraction with varying decimal markers.

### CODE REVIEW

File: test/unittests/test_parse_en.py [hunk 1]
```diff
@@ -290,6 +290,17 @@ def test_combinations(self):
 
 
 class TestExtractNumber(unittest.TestCase):
+    def test_extract_number_decimal_markers(self):
```
Comment (NeonDaniel) [inline test/unittests/test_parse_en.py:293]: Should include a test case for `1,000,000` and `1,000.0` (or similar). Maybe also `"1000, is what I said"` and/or `"I said 1,000."` to include boundary cases

**Review by NeonDaniel — CHANGES_REQUESTED** [review]
Couple test cases to add

**Comment by JarbasAl** [comment]
@coderabbitai review
